app/index.html

```diff
@@ -1,15 +1,13 @@
 <!DOCTYPE html>
 
 <head>
-	<meta charset='utf-8'>
-	<link rel='stylesheet' href='./css/index.css'>
+	<meta charset="utf-8" />
+	<link rel="stylesheet" href="./css/index.css" />
```

Comment:
Why did these get switched to double quotes?  Is the intent to change the style, or was this an accident?  IF it was intentional, it should be in a separate PR.

Reply:
the editor automatically formats it to double quotes when saving even if prettier is already set to single quotes. amended

Reply:
Hmm, this suggests you have some editor setting configured.  I am able to save `.html` files in VSCode without it converting `'` to `"` for me.